templates/default.nginx_http.security.conf.template

```diff
@@ -27,3 +27,7 @@ proxy_hide_header Server;
 add_header X-Content-Type-Options "nosniff" always;
 
 add_header Strict-Transport-Security $hsts_header always;
+
+{{^PRODUCTION}}
+add_header X-Robots-Tag "noindex, nofollow" always;
```

Comment:
This is going to add the header for all responses, including API responses. Ideally we would only add it for HTML responses, although it doesn't hurt. Optional

Reply:
Will leave that to be addressed with https://ucsc-cgl.atlassian.net/browse/DOCK-1754